charts/argocd-image-updater/templates/deployment.yaml

```diff
@@ -84,13 +84,30 @@ spec:
             name: registries-conf
           - mountPath: /tmp
             name: tmp-dir
+          {{- if .Values.authScripts.enabled }}
+          {{- range $k, $v := .Values.authScripts.scripts }}
+          - mountPath: /scripts
+            name: {{ $k | replace "." "-" }}
+          {{- end }}
+          {{- end }}
       volumes:
-      - configMap:
+      - name: registries-conf
```

Comment:
Could you please restore the alphabetical order and leave the `name:` attribute where it was?